apps/ledger-live-mobile/fastlane/Fastfile

```diff
@@ -427,15 +431,13 @@ platform :android do
     gradle(task: "clean", project_dir: 'android/')
     if (options[:local])
       gradle(
-        task: task,
-        build_type: type,
```

Comment:
You don't use type anymore, should it be removed above ?

Reply:
yes we do here `tasks = tasks.map { |task| task + type }`

Reply:
Oh nice my bad, I didn't pay enough attention